translation.rst

```diff
@@ -467,6 +467,26 @@ The ``translation:extract`` command looks for missing translations in:
 
         $ composer require nikic/php-parser
 
+By default, the ``translation:extract`` command extracts new messages in
+*message/translation* pairs, with the translation populated with the same
+text as the message, prefixed by ``__``. You can customize this prefix using
+the ``--prefix`` option.
+
+If you want to leave new translations empty, use the ``--no-fill`` option.
```

Comment:
Could you please add a `versionadded` directive for this new option ?

Reply:
Done. I put directive after example.